pkgs/tools/typesetting/tex/texlive/generate-fixed-hashes.nix

```diff
@@ -5,7 +5,7 @@ with lib; let
 
   # ugly hack to extract combine from collection-latexextra, since it is masked by texlive.combine
   combine = lib.findFirst (p: (lib.head p.pkgs).pname == "combine") { pkgs = [ ]; } (lib.head texlive.collection-latexextra.pkgs).tlDeps;
-  all = filter (p: p ? pkgs) (attrValues (removeAttrs texlive [ "bin" "combine" "combined" "tlpdb" ])) ++ [ combine ];
+  all = filter (p: p ? pkgs) (attrValues (removeAttrs texlive [ "bin" "combine" "combined" "tlpdb" ])) ++ [ { inherit (combine) pkgs; } ];
```

Comment:
Might as well get rid of that abomination now and use the internal interface `__pkgs`, this makes my eyes bleed.

Reply:
done!